app/assets/javascripts/logger.js

```diff
@@ -69,21 +69,24 @@ LoggerUtils.prototype._submittedQuestionLogging = function(data) {
 
 LoggerUtils.prototype._interactiveSimulationLogging = function() {
   // work around for bindig click event on cross site iframe
-  var self  = this,
-  myConfObj = {iframeMouseOver: false};
+  // FIXME: this code no longer works, as #interactive has been removed (now .interactive class is available instead
+  //        and we should assume that there can be multiple interactives on a single page).
```

Comment:
Yes, it seems it's been fixed, comment removed in https://github.com/concord-consortium/lara/commit/32d9638a6f125c561a20f36c8f7320ca180578e7.